b2luigi/batch/processes/lsf.py

```diff
@@ -30,17 +30,23 @@ class LSFProcess(BatchProcess):
     """
     Reference implementation of the batch process for a LSF batch system.
 
-    Additional to the basic batch setup (see :ref:`batch-label`), additional
-    LSF-specific things are:
+    Additional to the basic batch setup (see :ref:`batch-label`), there are
+    LSF-specific settings. These are:
 
-    * the LSF queue can be controlled via the ``queue`` parameter, e.g.
+    * the LSF queue: ``queue``.
+    * the LSF job name: ``job_name``.
+
+    Both can be controlled via the standard settings interface, e.g.
```

Comment:
I think it's not necessary to repeat that these can be controlled via settings, because in line 34 you already mention that these are settings and in the b2luigi documentation it should be clear that b2luigi settings are meant. So I would remove this sentence or replace it by a simple _For example:_
```suggestion
    For example:
```